.github/workflows/sast.yml

```
@@ -36,10 +36,10 @@ jobs:
       with:
         languages: ${{ matrix.language }}
 
-    - name: Setup JDK 8
+    - name: Setup JDK 17
       uses: actions/setup-java@v2
       with:
-          java-version: 8
+          java-version: 17
```

yamllint

[warning] wrong indentation: expected 8 but found 10

(indentation)